packages/app/cypress/support/commands.js

```diff
@@ -17,3 +17,56 @@ Cypress.Commands.add('enterAsGuest', () => {
   cy.visit('/');
   cy.get('button').contains('Enter').click();
 });
+
+Cypress.Commands.add('login', () => {
+  window.localStorage.setItem('@backstage/core:SignInPage:provider', 'guest');
+});
+
+Cypress.Commands.add('getTechDocsShadowRoot', () => {
+  cy.get('[data-testid="techdocs-content-shadowroot"]').shadow();
+});
+
+Cypress.Commands.add('mockTechDocs', () => {
+  cy.intercept(
+    'GET',
+    '**/techdocs/metadata/entity/default/Component/backstage',
+    {
+      fixture: 'techdocs/metadata-entity.json',
+    },
+  );
+
+  cy.intercept(
+    'GET',
+    '**/techdocs/metadata/techdocs/default/Component/backstage',
+    {
+      fixture: 'techdocs/metadata-techdocs.json',
+    },
+  );
+
+  cy.intercept('GET', '**/techdocs/sync/default/Component/backstage', {
+    fixture: 'techdocs/sync.json',
+  });
+
+  // Backstage Roadmap - TechDocs HTML
+  cy.intercept(
+    'GET',
+    '**/techdocs/static/docs/default/Component/backstage/overview/roadmap/index.html',
+    {
+      fixture: 'techdocs/components/roadmap.html',
+    },
+  );
+
+  // Backstage Home - TechDocs HTML
+  cy.intercept(
+    'GET',
+    '**/techdocs/static/docs/default/Component/backstage/index.html',
+    {
+      fixture: 'techdocs/components/default.html',
+    },
+  );
+
+  // TechDocs CSS
+  cy.intercept('GET', '**/assets/stylesheets/main.fe0cca5b.min.css', {
```

Comment:
Don't think we can mock this out as it will change hash every time there's a new release. What's the purpose of mocking this stylesheet?

Reply:
Yes you are right! I forgot to remove this mock when I removed the fixtures.